### PR TITLE
Implement list files at path

### PR DESCRIPTION
**Pull request by adam-tylr**
Issue #186

### CODE REVIEW

**Comment by adam-tylr** [comment]
@cdancy Can you create a release once this is approved?

**Comment by cdancy** [comment]
> Can you create a release once this is approved?

@adam-tylr you got it.

File: src/main/java/com/cdancy/bitbucket/rest/features/FileApi.java [hunk 1]
```diff
@@ -94,14 +94,16 @@ Commit updateContent(@PathParam("project") String project,
                          @Nullable @FormParam("sourceBranch") String sourceBranch);
 
     @Named("file:list-files")
-    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/5.1.0/bitbucket-rest.html#idm45588159557712"})
+    @RequestFilters(ScrubNullFromPathFilter.class)
```
Comment (cdancy) [inline src/main/java/com/cdancy/bitbucket/rest/features/FileApi.java]: I see this is coming in handy :)

Comment (adam-tylr) [reply src/main/java/com/cdancy/bitbucket/rest/features/FileApi.java]: Yes! There are several APIs following that format

**Comment by cdancy** [comment]
LGTM and works as expected. Thanks again @adam-tylr !

**Comment by cdancy** [comment]
@adam-tylr `2.5.1` has been released.

**Pull request merged**